### DISCUSSION TITLE
How do I compare the key press?

### DISCUSSION THREAD

**Discussion by samuelh2005** [byteduck/duckOS#4] [Q&A] [answered]
Ok so, I'm making some king of login screen and want to detect the Control+Alt+Delete sequence. So far I've got the control and alt keys detect, but now, how to check for the delete key?
```c++
bool LogonWidget::on_keyboard(Pond::KeyEvent evt){
	bool control = KBD_MOD_CTRL && evt.modifiers;
	bool alt = KBD_MOD_ALT && evt.modifiers;
	if(control && alt){
		evt.key;//How do I check if this is the delete key
	}
	return false;
}
```

**Comment by byteduck** [comment, marked as answer]
I believe the keycode for delete is 46 - you can see a list of standard keycodes [here](https://asawicki.info/nosense/doc/devices/keyboard/key_codes.html). Also, make sure you're using bitwise `&` for your control and alt - something like `KBD_MOD_CTRL & evt.modifiers`. `evt.modifiers` is a bitfield of all the different keyboard modifiers, and `KBD_MOD_CTRL` and `KBD_MOD_ALT` are all single bits corresponding to the appropriate modifier.
